README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  Alpha Merchant App - Simple Backend
+</p>
+
+
+## Description
+
+Simple backend to demonstrate how to implement the endpoints to integrate with Trustly.
+
+## Clone the app
+
+```bash
+# clone the repository
+$ git clone https://github.com/TrustlyInc/trustly-alpha-merchant-backend.git
+```
+
+## Configuring Trustly API credentials
+Create `.development.env` file in `trustly-alpha-merchant-backend` app folder. In this file we need to declare the environment variables:
```

Comment:
Can we create this file as a sample with dummy values and then the user will just have to input their own values?